packages/gen-ai/frontend/src/app/Chatbot/ChatbotMain.tsx

```diff
@@ -1,203 +1,149 @@
-            onClick={() => setIsViewCodeModalOpen(true)}
-          >
-            View Code
-          </Button>
-        ))
-      }
-    >
-      <ChatbotSourceSettingsModal
-        isOpen={sourceManagement.isSourceSettingsOpen}
-        onToggle={() =>
-          sourceManagement.setIsSourceSettingsOpen(!sourceManagement.isSourceSettingsOpen)
+          ))
         }
-        onSubmitSettings={sourceManagement.handleSourceSettingsSubmit}
-      />
-      <ViewCodeModal
-        isOpen={isViewCodeModalOpen}
-        onToggle={() => setIsViewCodeModalOpen(!isViewCodeModalOpen)}
-        input={input}
-        model={modelId}
-        systemInstruction={systemInstruction}
-      />
-      <Drawer isExpanded isInline position="right">
-        <DrawerContent panelContent={settingsPanelContent}>
-          <DrawerContentBody>
-            <Chatbot displayMode={displayMode} data-testid="chatbot">
-              <ChatbotContent>
-                <MessageBox position="bottom">
-                  <ChatbotWelcomePrompt
-                    title={username ? `Hello, ${username}` : 'Hello'}
-                    description="Welcome to the chat playground"
-                  />
-                  <ChatbotMessages
-                    messageList={chatbotMessages.messages}
-                    scrollRef={chatbotMessages.scrollToBottomRef}
-                  />
-                </MessageBox>
-              </ChatbotContent>
-              <ChatbotFooter>
-                <MessageBar
-                  onSendMessage={(message) => {
-                    if (typeof message === 'string') {
-                      chatbotMessages.handleMessageSend(message);
-                      setInput(message);
-                    }
-                  }}
-                  hasAttachButton={false}
-                  isSendButtonDisabled={chatbotMessages.isMessageSendButtonDisabled}
-                  data-testid="chatbot-message-bar"
-                />
-                <ChatbotFootnote {...{ label: 'Bot uses AI. Check for mistakes.' }} />
-              </ChatbotFooter>
-            </Chatbot>
-          </DrawerContentBody>
-        </DrawerContent>
-      </Drawer>
-    </ApplicationsPage>
+      >
+        {lsdStatus?.phase === 'Ready' ? (
+          <ChatbotPlayground
+            isViewCodeModalOpen={isViewCodeModalOpen}
+            setIsViewCodeModalOpen={setIsViewCodeModalOpen}
+          />
+        ) : lsdStatus?.phase === 'Failed' ? (
+          <EmptyState
+            headingLevel="h4"
+            titleText="Failed to configure playground"
+            variant={EmptyStateVariant.lg}
+            status="danger"
+          >
+            <EmptyStateBody>Please delete the playground and try again</EmptyStateBody>
+            <EmptyStateFooter>
+              <Button
+                variant="primary"
+                onClick={() => {
+                  if (namespace?.name) {
+                    deleteLSD(namespace.name, lsdStatus.name).then(refresh);
```

Comment:
The delete doesn't work here because the BFF is looking for the `displayName` of the LSD to delete, which is wrong, we should always look for the metadata.name to delete because displayName could be duplicated. I already notified @varshaprasad96 about that.